content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -45,7 +45,7 @@ uses a service of type [Service.Type=NodePort](/docs/concepts/services-networkin
 
 {{< feature-state for_k8s_version="v1.1" state="beta" >}}
 
-Before you start using an Ingress, there are a few things you should understand. The Ingress is a beta resource. 
+Before you start using an Ingress, there are a few things you should understand. The Ingress is a beta resource.
```

Comment:
I'm assuming that Ingress has left beta; we're long past v1.1. Is that right?

Reply:
This needs to be confirmed before it's updated in the document. Also, I don't think it is related to the current. We can open another bug to track it.